core/src/module/usModuleEvent.cpp

```diff
@@ -99,10 +99,12 @@ std::ostream& operator<<(std::ostream& os, ModuleEvent::Type eventType)
 {
   switch (eventType)
   {
-  case ModuleEvent::LOADED:    return os << "LOADED";
-  case ModuleEvent::UNLOADED:  return os << "UNLOADED";
-  case ModuleEvent::LOADING:   return os << "LOADING";
-  case ModuleEvent::UNLOADING: return os << "UNLOADING";
+  case ModuleEvent::LOADED:         return os << "LOADED";
```

Comment:
Clients can still start and stop bundles. LOADED/LOADING equates to STARTED/STARTING and UNLOADING/UNLOADED equates to STOPPED/STOPPING now. These are all valid bundle states given the current functionality of CppMicroServices.

Nomenclature changes will make this clearer.